packages/components/src/components/file/file.component.ts

```diff
@@ -300,6 +301,56 @@ export default class SynFile extends SynergyElement implements SynergyFormContro
     this.files = files;
   }
 
+  private async handleTransferItems(items: DataTransferItemList | null): Promise<FileList> {
+    if (!items || items.length !== 1) {
+      this.value = '';
+      return new Promise((_resolve, reject) => { reject(new Error('No proper items found')); });
+    }
+
+    const itemsArray = Array.from(items);
+
+    const entries: (FileSystemEntry | null)[] = itemsArray.map((item) => item.webkitGetAsEntry());
+
+    if (!entries[0]?.isDirectory) {
```

Comment:
Because I don´t have a clue: Can we be sure that a directory entry is always position zero? What if a user drags three images and a directory at the same time? Don´t know if there are predefined settings for this.